src/main/java/seedu/address/model/Model.java

```diff
@@ -237,6 +240,53 @@
      */
     void updateSortedTodoList(Comparator<Task> comparator);
 
+    // ========================== Scheduler Methods ============================================ //
+
+    /**
+     * Replaces Event list data with the data in {@code Eventlist}.
+     */
+    void setEventList(ReadOnlyEventList eventList);
+
+    /** Returns the EventList */
+    ReadOnlyEventList getEventList();
+
+    /**
+     * Returns true if a Event with the same identity as {@code Event} exists in the Event list.
```

Comment:
Minor error. Maybe Event should be event instead of capitalised? and should be an event instead of a event.

Reply:
Im referring the to Event object so it should be